templates/services/portainer/portainer_autorun

```diff
@@ -0,0 +1,6 @@
+portainer_autorun=true
+
+if [ "portainer_autorun" = true ]; then
+  docker-compose -f /srv/portainer/portainer.yml -p portainer up -d
```

Comment:
ooops @kjong need change this or the `docker` above

Reply:
updated